### ISSUE TITLE
Set a process for downloading all clearinghouse documents in an ongoing basis

### ISSUE THREAD

**Issue by bpdesigns** [18F/FAC-Distiller#78]
- all agency docs
- all pdfs

**Comment by bpdesigns** [comment]
Hi @danielnaab I'm closing this ticket as it was merged. Please let me know if its actually still a work in progress. Thanksd.

**Comment by bpdesigns** [comment]
from @danielnaab > The daily table load is still failing when run in the web container. Running it in that manner isn’t a good idea anyway, so I think spending a few hours to try and get CircleCI spinning the job up would be the way to go.

**Comment by cantsin** [comment]
Yes, we will have to restructure our periodic tasks to run somewhere else -- cloud.gov tasks aren't powerful enough for everything we want to do. There's many ways in which we can do so, but this is infrastructure work and probably beyond the scope of any work at this point. 

If we do go with CircleCI, I'd estimate two to three days of solid work for a engineer to 

1. understand the current scraping task and overall structure 
2. separate scraping from loading 
3. move over to CircleCI (using a cron scheduler) for scraping
4. figure out IAM permissions to write to S3 from CircleCI
5. update FAC distiller task to load only

Please note that I am making a big assumption here: that loading FAC documents from S3 (and nothing else) can still be done in a cloud.gov task -- I believe this is relatively low processing (compared to scraping) but if loading is still a problem, we may have to give up on cloud.gov tasks entirely.

**Comment by danielnaab** [comment]
Hi @cantsin - My thinking was that cloud.gov tasks can continue be used, but they would get spawned by a CircleCI job. The current S3 crawls and tables loads have been run in cloud.gov tasks successfully, but they were spawned manually.

**Comment by cantsin** [comment]
:wave: Got it! Then yes, that simplifies things somewhat. New approach:

1. Remove `apscheduler` cron code in FAC distiller
2. Set up CircleCI (cron scheduler) to invoke cloud.gov tasks (will require permission finagling -- creds should only target FAC distiller and nothing else)

If above is correct, then likely one to two days of work. @danielnaab please let me know if I've forgotten or missed something here. Thanks!
